simpleimage/make_rootfs.sh

```diff
@@ -390,6 +423,86 @@ EOF
 		rm -f "$DEST"/etc/ssh/ssh_host_*
 		do_chroot ln -s /run/resolvconf/resolv.conf /etc/resolv.conf
 		;;
+	opensuse|opensuse-lcd)
+		cat > "$DEST/etc/resolv.conf" << "EOF"
+nameserver 8.8.8.8
+nameserver 8.8.4.4
+EOF
+
+		cat > "$DEST/second-phase" <<EOF
+#!/bin/sh
+zypper remove -y kernel-default kernel-firmware
+zypper update
+zypper install -y dosfstools rfkill newt xterm patterns-openSUSE-yast2_basis
+zypper ar -f http://files.pine64.org/opensuse/repository "Pine64 support packages for openSUSE"
+echo
+echo "###########################################################################"
+echo "# For the prompt below, please answer it with 'a' for optimum performance #"
```

Comment:
Why would it ever prompt? Please do not introduce changes which require interaction.